package.json

```diff
@@ -13,41 +13,46 @@
     "@gluestack-style/react": "^1.0.54",
     "@gluestack-ui/config": "^1.1.18",
     "@gluestack-ui/themed": "^1.1.22",
-    "@react-native-async-storage/async-storage": "^1.23.1",
-    "@react-native-community/datetimepicker": "^7.6.1",
+    "@react-native-async-storage/async-storage": "1.21.0",
+    "@react-native-community/datetimepicker": "7.6.1",
     "@react-native-picker/picker": "2.6.1",
     "@react-navigation/drawer": "^6.6.15",
     "@react-navigation/material-top-tabs": "^6.6.13",
     "@react-navigation/native": "^6.1.17",
     "@react-navigation/stack": "^6.3.29",
     "@reduxjs/toolkit": "^2.2.3",
     "@ui-kitten/components": "^5.3.1",
+    "axios": "^1.6.8",
     "crypto-js": "^4.2.0",
     "expo": "~50.0.17",
     "expo-constants": "~15.4.6",
+    "expo-file-system": "~16.0.9",
     "expo-font": "~11.10.3",
     "expo-image-picker": "~14.7.1",
     "expo-linking": "~6.2.2",
+    "expo-permissions": "^14.4.0",
     "expo-router": "~3.4.10",
     "expo-splash-screen": "~0.26.5",
     "expo-status-bar": "~1.11.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "openai": "^4.44.0",
```

Comment:
btw, dafa3et meshe l 7al?

Reply:
working on this issue, waiting for a new gmail account